plugin.cpp

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (C) 2016, Pelagicore AB.
+ * Author: Jonatan Pålsson <jonatan.palsson@pelagicore.com>
+ * SPDX-License-Identify: MPL-2.0
+ */
```

Comment:
Ah. I wasn't aware of this example file when I wrote my previous comment. If there is a standard within the company, then we should follow it. I would argue that this standard can be improved though (same arguments as my previous comment).